languages/pt.ini

```
@@ -301,6 +301,7 @@ Exception = "Excepção"
 Excerpt = "Excerto"
 exclude_facet = "[excluír]"
 exclude_newspapers = "Excluir os artigos de jornal"
+excluded = "Excluídos:"
```

```suggestion
excluded = "Excluído:"
```
Updating translation per suggestion from @becahp - thanks, Rebeca!